addon/-private/environment/data.js

```diff
@@ -10,7 +10,7 @@ const {
  * Wraps an object, resolving any Bindings in it when requested to the corresponding
  * paths in the given environment.
  */
-export default class EnvironmentData extends EmberObject {
+export default class EnvironmentData extends EmberObject.extend(Ember.Evented) {
```

Comment:
Should `EnvironmentData` need to be evented?

Reply:
I thought it would but turns out no